### ISSUE TITLE
Order Management - Order Management Redesign: Refund Functionality

### ISSUE THREAD

**Issue by shanimal08** [big-neon/bn-web#1513]
- Org admin or higher should be able to take refund actions
  - From the Event Summary Card
    - User should be able to complete a Full Refund of the order total less any previous refunds
  - From the Expanded View
    - If an item is already refunded it should not be selectable
    - If a single or multiple ticket line item is selected:
      - A bottom action bar should appear with a CTA to refund the value of the selected items

**Comment by Anziskavds** [comment]
@Jasonvdb Is this correct? It's a lot different from the designs on Abstract?  ![image.png](https://images.zenhubusercontent.com/5cdd14f21f980125322a448a/8273f0c7-8cce-4dee-8eec-1561931891a7)

@Tas4tari

**Comment by Jasonvdb** [comment]
Yes it'll be a little different for now as we're not supporting all the different ways a refund can happen like the designs specified

**Comment by Anziskavds** [comment]
Tested on develop and found that the refund successful screen does only pop up for a second or so and disappears again. There should be a refund successful screen pop up after you clicked on **confirm**. Please have a look at this video and look at the middle of the screen after I clicked confirm, you'll see it popped up and disappeared again after a sec or so.

https://cl.ly/b6c4451b4d11

**Comment by Jasonvdb** [comment]
@Anziskavds ok I see what's happening here, we don't actually have a refund success screen yet, it's just that green success message at the bottom. What you're seeing in that flash is the refund dialog that's quickly resetting before it closes. 

Will work on a refund success screen now.

**Comment by Jasonvdb** [comment]
@Anziskavds please check again on develop, have added a success screen. Test with both full refunds and with selecting individual tickets refunds

**Comment by Anziskavds** [comment]
Tested develop and it tested successful.

**Comment by Anziskavds** [comment]
Tested successful on Beta for web and mobile web.
